src/main/java/endpointurl/Element.java

```diff
@@ -0,0 +1,5 @@
+package endpointurl;
+
+public class Element {
+    public static final String INDIVIDUAL = "Individual";
+}
```

Comment:
You have many classes for static variables. You need to organize better those variables